ironic_bug_dashboard/__init__.py

```diff
@@ -81,6 +81,18 @@ async def index(request):
     ironic_bugs['all'] = [x for x in ironic_bugs['all']
                           if x['importance'] != 'Wishlist']
 
+    nova_triaged_bugs = []
+    nova_unconfirmed_undecided = []
+    for bug in nova_bugs['all']:
+        if (bug['status'] == 'Confirmed' and bug['importance'] != 'Undecided'):
```

Comment:
Confirmed is not the only triaged status in launchpad. You also get Triaged, In Progress, etc. I think it should be the other way around: if the status is "New" OR importance is "undecided", then it's not triaged.